docker-compose.dev.yml

```diff
@@ -10,7 +10,7 @@ services:
       SERVER_DOMAIN: ${SERVER_DOMAIN}
       IMGUR_CLIENT_ID: ${IMGUR_CLIENT_ID}
     volumes:
-      - $PWD/caddy/Caddyfile.dev:/etc/caddy/Caddyfile
+      - ./caddy/Caddyfile.dev:/etc/caddy/Caddyfile
```

Comment:
And this change doesn't have to be here either.

Reply:
yeah - rigth